doc/faq.rst

```diff
@@ -34,12 +34,15 @@ An ECC key is roughly 500 bytes formatted in Base64 and RSA 2048 key is
 we opted for readability.
 
 
-Why do you drop all headers if there is more than one?
-------------------------------------------------------
-
-Because of multi-agent usage we may have to handle an inconsistent stream of
-headers already. Making this an inconsistent stream of multiple keys with
-priorities sounds like a lot of pain.
+Why do you ask to drop all headers if there is more than one?
+-------------------------------------------------------------
+
+Because of multi-agent usage we may have to handle an inconsistent
+stream of headers already. Making this an inconsistent stream of
+multiple headers per mail sounds like a lot of pain. We could come up
```

Comment:
I think this is a Postel's law thing. If we receive two valid autocrypt headers, clearly *something* must be going on. If we are "liberal in what we accept" here, e.g. by choosing one at random, that means clients can get away with breaking the MUST NOT in regard to sending multiple headers that can be treated as equally valid (while we have no defined order of preference). If we do that, we might as well soften that MUST criteria.